public/app/features/explore/Wrapper.test.tsx

```diff
@@ -261,12 +266,36 @@ describe('Wrapper', () => {
       // to work
       await screen.findByText(`loki Editor input: { label="value"}`);
 
-      store.dispatch(splitOpen<any>({ datasourceUid: 'elastic', query: { expr: 'error' } }) as any);
+      store.dispatch(mainState.splitOpen<any>({ datasourceUid: 'elastic', query: { expr: 'error' } }) as any);
 
       // Editor renders the new query
       await screen.findByText(`elastic Editor input: error`);
       await screen.findByText(`loki Editor input: { label="value"}`);
     });
+
+    it('handles split size events and sets relevant variables', async () => {
```

Comment:
@ifrost @Elfo404 Here is the test I arrived to. I wanted to test pane width but it doesn't seem to be easy to do - the element that gets a CSS width is the autosizer, which is mocked in this to always return a div of the same size. All the other elements I can select return 0 as the width.